<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMultipleProfileAccountsDialog</class>
 <widget class="QDialog" name="EditMultipleProfileAccountsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit information of selected accounts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="serverLocationLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Server location:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="serverLocationComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>International / English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spanish</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="serverLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="channelComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Channel 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 9</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="channelLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="serverComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="characterLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Character:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="characterComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Stay at character selection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Character 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Character 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Character 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Character 4</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="loginCheckBox">
     <property name="text">
      <string>Automatically log in selected server and channel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addProfAccountButton">
     <property name="text">
      <string>Edit accounts</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
